wnprc_billing/resources/queries/ehrLinked/project.query.xml

```diff
@@ -8,6 +8,7 @@
                             <fkDbSchema>ehrLinked</fkDbSchema>
                             <fkTable>investigators</fkTable>
                             <fkColumnName>rowid</fkColumnName>
+                            <fkDisplayColumnName>lastName</fkDisplayColumnName>
```

Comment:
Do we need this in addition to the titleColumn in investigators.query.xml

Reply:
this now points to a new investigatorsWithName query.